lib/ansible/plugins/loader.py

```diff
@@ -52,10 +54,19 @@
 get_with_context_result = namedtuple('get_with_context_result', ['object', 'plugin_load_context'])
 
 
-def get_all_plugin_loaders():
+@functools.cache
+def get_all_plugin_loaders() -> list[tuple[str, 'PluginLoader']]:
```

Comment:
Did you have to comment `'PluginLoader'` here? I would have thought `__future__ import annotations` would have allowed just the plain `PluginLoader`.